src/Uno.UI/UI/Xaml/Documents/Hyperlink.cs

```diff
@@ -148,22 +151,45 @@ private void OnUnderlineStyleChanged()
 
 		#endregion
 
+		#region Hover
+		private Pointer _hoveredState;
+		internal void SetPointerOver(Pointer pointer)
+		{
+			_hoveredState = pointer;
+			SetCurrentForeground();
+		}
+
+		internal bool ReleasePointerOver(Pointer pointer)
+		{
+			if (_hoveredState?.Equals(pointer) ?? false)
+			{
+				_hoveredState = null;
+				SetCurrentForeground();
+				return true;
+			}
+			else
+			{
+				return false;
+			}
+		}
+		#endregion
+
 		#region Click
-		private Pointer _pressedPointer;
+		private Pointer _pressedState;
```

Comment:
The old name `_pressedPointer` should be good now that this no longer holds the resources.